src/Module/Xrd.php

```diff
@@ -65,13 +65,21 @@ protected function rawContent(array $request = [])
 
 		if (substr($uri, 0, 4) === 'http') {
 			$name = ltrim(basename($uri), '~');
+			$parse = parse_url($uri);
+			$host = $parse['host'];
```

Comment:
These two lines can be simplified to `parse_url($url, PHP_URL_HOST)`

Reply:
My first steps with php ....

Reply:
You're doing great!